modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -1,101 +1,127 @@
+i.e the displacements blow-up as shown by the asymptopes in [cantileverfrf].
+
+!media media/tensor_mechanics/Cantilever_frfmag.png style=width:60%; caption=Cantilever beam displacement magnitude for coarse mesh (0.1m elements) and fine mesh (0.033m elements).  Analytic results are shown in grey. id=cantileverfrfmag
 
-!alert note
-$u$ is a function of time and space $(x,t)$, whereas $U$ is function of space and frequency $(x,\omega)$.
+!media media/tensor_mechanics/Cantilever_frf.png style=width:60%; caption=Cantilever beam y- and z-displacements for the fine mesh (0.033m elements).  Analytic results are shown in grey. id=cantileverfrf
 
-where $\hat \sigma = \nabla U$ with $U(x,\omega)$ being the Fourier transform of $u(x,t)$.
+The input file for the coarse simulation shown in [cantileverfrfmag] and [cantileverfrf] is given in [listing0].
 
-!alert note title=Fourier Transfrom of Boundary Conditions
-Not only the pde is transformed to the frequency domain, but also, we must transform all B.Cs as well. Transfroming $A cos(2\pi f t)$ into the frquency domain will result in an impulse at frequency = $f$ with amplitude of A.
+!listing examples/wave_propagation/cantilever_sweep.i id=listing0
 
-When transform the time dependent variable $u$ to the frequency domain, the time-dependency will be omitted. The result is a nother variable $U$ that is frequency dependent. In general, $U$ is complex-valued function/variable in the form of $U_r+iU_i$, where $U_r$ and $U_i$ are the real and the imaginary part of $U$.
+This uses the [TensorMechanics MasterAction](Modules/TensorMechanics/Master/index.md) to setup the StressDivergence Kernels because this simulation only includes real displacements.  A `Transient` executioner with a `ConstantDT` timestepper is used to sweep over frequencies where the time is substituted for the frequency.  The time is converted to the frequency dependent `Reaction` `rate` using a `ParsedFunction` and is applied to the `rate` using the `RealFunctionControl`.
 
-At this stage, we split the complex sytem of equations into two $\textbf{real}$ systems, which yields two real-valued systems that live on the same mesh. The strong form PDEs becomes as follows:
+In the next example, imaginary displacements will be introduced by including an absorbing boundary condition.  Imaginary displacement represent a phase shift in the problem and often result from damping.  
 
+# Uniaxial Beam Example
+
+The displacement field $U$ is a complex number.  In the previous example, only the real part of the displacement field was accounted for.  In this example, an absorbing boundary condition will introduce a coupling between the real and imaginary displacement fields.  This example uses the uniaxial beam shown in [cantilever] which is subjected to time-harmonic displacement on right and has an absorbing boundary condition on left.  The absorbing boundary condition is given by the Sommerfeld radiation condition as follows
```

Comment:
In the Fourier domain, the displacement field $U$ is in general a complex number.

Reply:
done